src/ledger/LedgerTxnImpl.h

```diff
@@ -376,9 +376,19 @@ class LedgerTxn::Impl::EntryIteratorImpl : public EntryIterator::AbstractImpl
 // been lost.
 class LedgerTxnRoot::Impl
 {
-    typedef std::string EntryCacheKey;
-    typedef cache::lru_cache<EntryCacheKey, std::shared_ptr<LedgerEntry const>>
-        EntryCache;
+    enum class LoadType
```

Comment:
I don't think we're currently using these for anything but I think they are intended to facilitate the calculation of some metrics. Can we use these to design some metrics about how successful the prefetching strategy is?

Reply:
Thanks for the review! I'm out of time today, but I pushed a few commits that address your comments, except this one (which I'll re-visit when I get back).